apps/studio/components/interfaces/Auth/Policies/AIPolicyEditorPanel/PolicyDetailsV2.tsx

```diff
@@ -131,63 +131,11 @@ export const PolicyDetailsV2 = ({
                   </p>
                 </FormLabel_Shadcn_>
                 <FormControl_Shadcn_>
-                  <Popover_Shadcn_ open={open} onOpenChange={setOpen} modal={false}>
```

Comment:
hold on this component is used in multiple places (primarily in the auth/policies) page - the changes here will affect the UI wherever this component is used and not just for the realtime inspector

Reply:
oh god, thanks Joshen! That was a giant misread on my part!